packages/manager/apps/web-office/src/api/license/api.ts

```diff
@@ -35,5 +35,12 @@ export const getOfficePrepaidLicenses = async (serviceName: string) => {
 // POST
 
 // PUT
+export const updateOfficeLicenseDetails = async (
+  serviceName: string,
+  licenseData: Partial<LicenseType>,
+): Promise<LicenseType> => {
+  const { data } = await v6.put(`${getApiPath(serviceName)}`, licenseData);
```

Comment:
string templating is useless here you can just put `v6.put(getApiPath(serviceName), licenseData)`